tools/lkl/lib/virtio_net_tap.c

```diff
@@ -122,5 +196,24 @@ struct lkl_netdev *lkl_netdev_tap_create(const char *ifname)
 		return NULL;
 	}
 
+	#ifdef CONFIG_EVENTFD
```

Comment:
Yes. Your understanding is correct. Feel free to invent any method. My baseline is we need support using eventfd and freeing up virtio at the same time.